README.md

```diff
@@ -14,21 +14,35 @@ This repository follows the [RealWorld](https://github.com/gothinkster/realworld
 
 ## Installation
 
-You can install and build the project with:
+Firstly, clone the repository:
 
 ```sh
-# clone the repository
 git clone https://github.com/thomashoneyman/purescript-halogen-realworld
 cd purescript-halogen-realworld
+```
+
+In order to install the JavaScript dependencies:
```

Comment:
`yarn install` will also run the `postinstall` script, which installs PureScript dependencies as well. Perhaps this could be reworded to

```suggestion
To install the JavaScript and PureScript dependencies:
```